src/airflow/dags/airnow_bam_data.py

```diff
@@ -8,7 +8,7 @@
     schedule=None,
     default_args=AirflowUtils.dag_default_configs(),
     catchup=False,
-    tags=["bam", "airnow", "historical"],
+    tags=["bam", "usembassy", "historical"],
```

Comment:
Was this the configuration to resolve the missing `usembassy` data? @BenjaminSsempala

Reply:
@Baalmart this was just to rename airnow to usembassy in the pipeline, just to reduce confusion, or improve readability

Reply:
Alright @BenjaminSsempala , thanks!